frontend/src/individual_pedigree.tsx

```diff
@@ -85,7 +84,8 @@ export function IndividualPedigree({ id, generations }: { id: string }) {
 
               </dl>
             </TableCell>
-            <TableCell width="85%" >
+            <TableCell width="85%" style={{textAlign: "left"}}>
+              <h3>Pedigree: <input value={generations_input} onChange={event => setGenerations(event.target.value)} type="number" min="1" max="50" /></h3>
               <PedigreeNetwork pedigree={pedigree} />
```

Comment:
Do we need to check that `pedigree !== undefined` first?

Reply:
Jonas suggested to render the individual data first, that is why I removed the validation, I added it now to the pedigree
```
{pedigree && <PedigreeNetwork pedigree={pedigree} />}
```